src/Microsoft.ML.LightGbm/LightGbmMulticlassTrainer.cs

```diff
@@ -197,13 +241,13 @@ private protected override void CheckAndUpdateParametersBeforeTraining(IChannel
             }
 
             if (useSoftmax)
-                Options["objective"] = "multiclass";
+                GbmOptions["objective"] = "multiclass";
             else
-                Options["objective"] = "multiclassova";
+                GbmOptions["objective"] = "multiclassova";
 
             // Add default metric.
-            if (!Options.ContainsKey("metric"))
-                Options["metric"] = "multi_error";
+            if (!GbmOptions.ContainsKey("metric"))
+                GbmOptions["metric"] = "multi_error";
```

Comment:
>"multi_error" [](start = 39, length = 13)

we don't use metric enum at all?

Reply:
Each trainer has its own respective metric enum and default metric type being assigned on initialization. 

---
In reply to: [265372586](https://github.com/dotnet/machinelearning/pull/2948#discussion_r265372586) [](ancestors = 265372586)